### PR TITLE
added blur-loading-page.html

### PR DESCRIPTION
**Pull request by nirban256**
Added Blur loading of background transition.

### CODE REVIEW

**Comment by nirban256** [comment]
@Metroxe please review the pull request

File: entries/blur-loading-page.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        height: 100vh;
+        overflow: hidden;
+        margin: 0;
+    }
+
+    .bg {
+        background: url('https://images.unsplash.com/photo-1529619768328-e37af76c6fe5?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1287&q=80') no-repeat center center/cover;
```
Comment (Metroxe) [inline entries/blur-loading-page.html:27]: You cannot use an external file, everything must be in one html file only. You can still use the image if its encoded as a base64 inline

**Review by Metroxe — CHANGES_REQUESTED** [review]

File: entries/blur-loading-page.html (hunk 1 above)
Comment (nirban256) [reply entries/blur-loading-page.html:27]: ok I will try to fix that

**Comment by nirban256** [comment]
@Metroxe I have changed the image with a static colour, review it once

**Review by Metroxe — APPROVED** [review]
looks good!

**Pull request merged**